raspios/config-dev.env

```diff
@@ -0,0 +1,23 @@
+# IP address of the beacon.
+# Will also be used as:
+#   - default gateway for robots
+#   - IP address for COGIP tools to access cogip-server
+IP_ADDRESS_BEACON_WLAN0=${IP_ADDRESS_BEACON_WLAN0:-192.168.1.199}
+IP_ADDRESS_BEACON_ETH0=${IP_ADDRESS_BEACON_ETH0:-192.168.1.1}
+
+# IP address of robot 1
+IP_ADDRESS_ROBOT1_WLAN0=${IP_ADDRESS_ROBOT1_WLAN0:-192.168.1.101}
+IP_ADDRESS_ROBOT1_ETH0=${IP_ADDRESS_ROBOT1_ETH0:-192.168.1.111}
+
+# IP address of robot 2
+IP_ADDRESS_ROBOT2_WLAN0=${IP_ADDRESS_ROBOT2_WLAN0:-192.168.1.102}
+IP_ADDRESS_ROBOT2_ETH0=${IP_ADDRESS_ROBOT2_ETH0:-192.168.1.112}
+
+# IP address of basket
+IP_ADDRESS_BASKET_WLAN0=${IP_ADDRESS_BASKET_WLAN0:-192.168.1.110}
+IP_ADDRESS_BASKET_ETH0=${IP_ADDRESS_BASKET_ETH0:-192.168.1.120}
```

Comment:
to remove ? :thinking:

Reply:
Basket must be removed, but it is not the purpose of this branch.

Reply:
Done in another commit.